JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -60,14 +60,16 @@ public async Task DispatchMessage(SQSMessage message, CancellationToken cancella
             }
             catch (MessageFormatNotSupportedException ex)
             {
-                _log.LogTrace($"Didn't handle message [{message.Body ?? string.Empty}]. No serializer setup");
+                _logger.LogTrace("Could not handle message with Id '{MessageId}' because a deserialiser for the content is not configured. Message body: '{MessageBody}'.",
```

Comment:
Question of consistency of whether we should use US English in the log messages because we use US English for the code identifiers, i.e. _deserializer_ vs. _deserialiser_.